### PR TITLE
Testkit change for scopes and collections for both CBL-C and CBL-ios

### PR DESCRIPTION
**Pull request by abhayayay**
#### Fixes #.

- [ ] Ran `flake8`
- [ ] Ran `run_repo_tests.sh`

#### Changes proposed in this pull request:

- 
-

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -55,7 +55,7 @@ rsa==3.3
 s3transfer==0.0.1
 six==1.15.0
 termcolor==1.1.0
-troposphere==1.5.0
+troposphere==4.0.2
```
Comment (gilkal) [inline requirements.txt:58]: This dependency update should already be in master. You could try to fetch and merge

File: CBLClient/Collection.py [hunk 2]
```diff
@@ -0,0 +1,110 @@
+from CBLClient.Client import Client
```
Comment (gilkal) [inline CBLClient/Collection.py:1]: Could you run flake8 and run_repo_tests.sh, and check the review's boxes once they pass? 
It's easily done by running source setup.sh and then flake8 and sh run_repo_tests.sh

The Jenkins job fail if flake8 doesn't pass.

File: testsuites/CBLTester/CBL_Functional_tests/APITests/test_database.py [hunk 3]
```diff
@@ -416,3 +416,4 @@ def test_update_document(self):
         for doc_id in cbl_docs:
             assert cbl_docs[doc_id]["new_field"] == updated_body[doc_id]["new_field"],\
                 "Doc body doesn't match after update"
+
```
Comment (gilkal) [inline testsuites/CBLTester/CBL_Functional_tests/APITests/test_database.py:419]: Is it to fix a flake8 issue?

File: CBLClient/Collection.py (hunk 2 above)
Comment (abhayayay) [reply CBLClient/Collection.py:1]: ./CBLClient/Database.py:398:1: W293 blank line contains whitespace
./CBLClient/Database.py:412:1: W293 blank line contains whitespace

will this be considered a flake8 error?
for repo_run_tests.sh there is no error.

File: testsuites/CBLTester/CBL_Functional_tests/APITests/test_database.py (hunk 3 above)
Comment (abhayayay) [reply testsuites/CBLTester/CBL_Functional_tests/APITests/test_database.py]: yes

**Pull request closed without merging**